automation/project/Dependencies.scala

```diff
@@ -7,7 +7,7 @@ object Dependencies {
   val akkaHttpV     = "10.2.0"
   val jacksonV      = "2.14.1"
 
-  val workbenchLibsHash = "1174fb6"
+  val workbenchLibsHash = "1123813"
```

Comment:
Update of workbench libs for the ability to disable WDS via the AttributeMap when calling `Rawls.workspaces.create`.

Reply:
Note though that this hash is specific to serviceTest (which is now a 2.1 prefix). I'm still trying to sort out how there is meant to be a single hash for all the different libraries.